apps/core/tests/test_management_commands_create_cms_data.py

```diff
@@ -93,6 +94,18 @@ def test_command_create_cms_data(
                     "is_homepage": False,
                     "published": True,
                     "site": site,
+                    "reverse_id": COURSES_PAGE_REVERSE_ID,
```

Comment:
For the test to be efficient, you should not import COURSES_PAGE_REVERSE_ID but key it manually to "courses".
So that if we change it unintentionally the test can save us (act like a confirmation, are you sure you want to change this because here is the impact...)

Same for ORGANIZATIONS_PAGE_REVERSE_ID and COURSE_SUBJECTS_PAGE_REVERSE_ID

Reply:
right !